### DISCUSSION TITLE
I ran the MAUI Blazor project on 7 different devices and took a video.

### DISCUSSION THREAD

**Discussion by aokocax** [dotnet/maui#5268] [Show and tell]
Today I ran the MAUI Blazor project on 7 different devices without any problems and I took a video. I also want to thank everyone who contributed here. In my opinion the best thing in a long time developed by Microsoft is undoubtedly MAUI.

Congratulations.

https://user-images.githubusercontent.com/1059475/158075701-757e943b-1547-4d0e-af4b-da7251a98d0c.mp4

Sorry for video quality .

**Comment by coderbasket** [comment]
There is a library here... with enums. css styling in c# and some cal().
each css enums has documentation and example along with various posible style values.
[https://github.com/coderbasket/StylerCode.Css](https://github.com/coderbasket/StylerCode.Css)

![alt text](https://raw.githubusercontent.com/coderbasket/StylerCode.Css/main/20220319_191849.jpg)

**Comment by mcguu** [comment]
Thank you  I want to do that too but I have only two device
sad

**Comment by mcguu** [comment]
mac windows ios android and ?
